Package/iOS/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.13.2)
 project(ReactNativeBabylon)
+
+# Clang will by default emit objc_msgSend stubs in Xcode 14, which ld from earlier Xcodes doesn't understand.
+# We disable this by passing -fno-objc-msgsend-selector-stubs to clang.
```

Comment:
Maybe a pointer to this link for additional info: https://stackoverflow.com/questions/73983452/xcode-14-xcode-13-incompatibility